### ISSUE TITLE
AttributeError when following doc example

### ISSUE THREAD

**Issue by FabioRosado** [prkumar/uplink#51]
When creating the code for the Pull Request #50  I've also noticed that when I try to add `.execute()` to the `.get_repos()` like such:

```
response = github.get_repos().execute()
```

The code fails with:

```
AttributeError: 'Response' object has no attribute 'execute'
```

Does the documentation need updating or do we need to implement the `execute()` function?

**Comment by prkumar** [comment]
The `execute` method was removed in [v0.2.0](https://github.com/prkumar/uplink/releases/tag/v0.2.0). `github.get_repos()` should directly return the response. In other words, this is the expected behavior. 

I tried to remove any references to `execute()` in the repository and the documentation, so let me know if you spot any lingering usages that I may have missed.

P.S.: Sorry for my delayed response to your PRs and Issues. I just got back from holiday vacation :)
